ruby_on_rails/rails_basics/controllers.md

```diff
@@ -227,4 +227,4 @@ This section contains questions for you to check your understanding of this less
 - <a class="knowledge-check-link" href="https://guides.rubyonrails.org/action_controller_overview.html#parameters">Which variable in your controller contains all the data sent in by the user?</a>
 - <a class="knowledge-check-link" href="#strong-parameters">Data from a client can never be trusted. Which helper method should you incorporate in your controller to prevent malicious data injections?</a>
 - <a class="knowledge-check-link" href="https://guides.rubyonrails.org/action_controller_overview.html#the-flash">When is a flash message shown?</a>
-- <a class="knowledge-check-link" href="https://guides.rubyonrails.org/action_controller_overview.html#flash-now">How do we change when a flash is shown?</a>
\ No newline at end of file
+- <a class="knowledge-check-link" href="https://guides.rubyonrails.org/action_controller_overview.html#flash-now">How do we change when a flash is shown?</a>
```

Comment:
I believe this change is due to removing the final blank line at the end of the file. If so, can you add this line back to the file because this final blank line is standard for all file types?

Reply:
Thanks for checking. Yes, that double-space would be great to remove.

Reply:
Sure, no problem.